### ISSUE TITLE
[1.8.8] Chunk load lag?

### ISSUE THREAD

**Issue by L4BORG** [TacoSpigot/TacoSpigot#45]
https://timings.aikar.co/v2/?id=52655da708fd4a0999d0c73e99b2deb1&section=lagsummary

any ideas what's going on?

**Comment by Techcable** [comment]
Sampling data please?

**Comment by melinstagibson** [comment]
This is an old issue that started with 1.9, you get it on spigot or paper too (me atleast) try to reduce the view distance to load fewer chunks or try to preload your map.
or find another way to discoourage exploring too much. maybe a far out random tp if you run a survival server.

btw its not a chunk load issue its a chunk generate issue. the server lags only when new chunks are generated by players

@Techcable mind explaining to me how you get sampling data and what it is just so i know in the future ? :D

**Comment by L4BORG** [comment]
@Techcable I would love to, but I can't run sampler without lagging server to death... we tried and talked about this before...

@melinstagibson how is this 1.9? It's hurting us on 1.8.8

Map is NOT pregenerated, would map pregeneration solve this?

I've tried a lot of things already and it didn't help, with view-distance 3 we can hold ~130 players without lag and with view-distance 4 ~90 players without lag... meanwhile Eggwars can hold 350 players without lag and with the same TacoSpigot and plugins. There is no map generation on EggWars tho... (plugin is (un)loading map and restoring files).
